deploy/auth-delegator.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
```

Comment:
If you want this to work on GKE I believe you need to use at least the v1beta1 version of core resources such as rbac.